runtime/jcl/common/mgmtmempool.c

```diff
@@ -504,6 +524,19 @@ getIndexFromPoolID(J9JavaLangManagementData *mgmt, UDATA id)
 	return idx;
 }
 
+static UDATA
+getIndexFromCollectorID(J9JavaLangManagementData *mgmt, UDATA id)
+{
+	UDATA idx = 0;
+
+	for(; idx < mgmt->supportedCollectors; ++idx) {
```

Comment:
Need space between for and bracket. Also I would provide initial value explicitly:
   for (idx=0; idx < mgmt->supportedCollectors; ++idx) {

Reply:
done